src/global_styling/variables/_typography.scss

```diff
@@ -37,8 +37,10 @@
 }
 
 // Families
-$ouiFontFamily: 'Inter UI', -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol' !default;
-$ouiCodeFontFamily: 'Roboto Mono', Consolas, Menlo, Courier, monospace !default;
+// sass-lint:disable quotes
+$ouiFontFamily: #{"'Inter UI', -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'"} !default;
+$ouiCodeFontFamily: #{"'Roboto Mono', Consolas, Menlo, Courier, monospace"} !default;
```

Comment:
What is the need for quotes here?

Reply:
Without those it would loose all quotes.